fsspec/utils.py

```diff
@@ -466,3 +467,12 @@ def wrapper(cls):
         return cls
 
     return wrapper
+
+
+try:
+    from contextlib import nullcontext
+except ImportError:
+
+    @contextmanager
```

Comment:
It's so simple, may as well just keep our own. That way, we can make the argument required.